### DISCUSSION TITLE
Embedding Stride in a UWP XAML app

### DISCUSSION THREAD

**Discussion by yoshiask** [stride3d/stride#1451] [Q&A]
Howdy! I'm trying to figure out how I might go about using Stride to draw 3D graphics in a UWP XAML app. Is this possible? If so, can you please point me to the relevant documentation, or perhaps a complete sample if available?

**Comment by Agredo** [comment]
Actually its pretty simple in UWP if you meant it like this https://github.com/xamarin/XamarinCommunityToolkit/issues/549

**Comment by DJDQfff** [comment]
4.1+ version can't export uwp

**Reply by Agredo** [reply]
This is beacause stirde bases since 4.1 on .NET 6. So you have to use Windows App SDK (WinUI3) as target.
